### DISCUSSION TITLE
Add consumer_group support into kong docker image

### DISCUSSION THREAD

**Discussion by josephwocken** [Kong/kong#11927] [Help]
I'm integrating with Kong for the first time and am wondering if ya'll could add support for `consumer_groups` into the `kong` [docker image](https://hub.docker.com/_/kong/) admin api? 

I'm developing a feature for api keys and have been able to test everything else locally except for this. I'd love to be able to fully test locally instead of bypassing this one thing. Here are the endpoints that I'd like to use:

1. `POST consumer_groups`
2. `POST consumer_groups/{consumer_group_id}/consumers`

**Comment by StarlightIbuki** [comment]
Hi. The consumer group is an enterprise feature. If you want to test with Kong Enterprise, you could get a trial version.

**Reply by josephwocken** [reply]
Thanks for the reply @StarlightIbuki ! We have an instance of Kong Enterprise already. We've setup runtime groups for our lower environments and production environment. I suppose we could setup a "local" environment runtime group on the Kong server, but it would be shared with all other developers. What I'd really like is to be able to test locally, on my computer.

I see the problem now though. If it's an enterprise-only feature it might not belong in the kong docker image, which covers public features only. Could be a nice feature for the docker image to run in "enterprise mode" with the extra features.
